src/BulkAppointments/AppointmentForm.js

```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
   }
 
   if (!values.appointmentType) {
-    formErrors.push({ targetName: 'appointmentType', text: 'Appointment type is required' })
+    formErrors.push({ targetName: 'appointmentType', text: 'Select appointment type' })
   }
 
   if (!values.location) {
-    formErrors.push({ targetName: 'location', text: 'Location is required' })
+    formErrors.push({ targetName: 'location', text: 'Select location' })
```

Comment:
I think `Select a location` would fit more with the GDS style messages, they have things like `Enter a postcode`

Reply:
The wording is from Clare. I'll catch up with her to see what she thinks